features/element-hiding.json

```diff
@@ -894,12 +928,29 @@
                     }
                 ]
             },
+            {
+                "domain": "tripadvisor.ca",
+                "rules": [
+                    {
+                        "selector": "#tripGoogleOnetapContainer",
+                        "type": "hide"
+                    },
+                    {
+                        "selector": ".mxEhR",
+                        "type": "hide"
+                    }
+                ]
+            },
             {
                 "domain": "tripadvisor.com",
                 "rules": [
                     {
                         "selector": "#tripGoogleOnetapContainer",
                         "type": "hide"
+                    },
+                    {
+                        "selector": ".mxEhR",
```

Comment:
This looks like it's a generated class that might change frequently, but I checked The Wayback Machine and it's been the same since at least December.